src/app/admin/messages/(components)/msg-detail.tsx

```
@@ -0,0 +1,175 @@
+        <motion.form layout={false} className="space-y-6" onSubmit={form.handleSubmit((data) => mutate(data))}>
+          <div className="space-y-4">
+            <h2 className="text-3xl font-bold text-gray-900 dark:text-white">{message.subject}</h2>
+            <div className="flex items-center space-x-4 text-sm text-gray-500 dark:text-gray-400">
+              <span className="flex items-center">
+                <svg
+                  className="mr-2 size-4"
+                  fill="none"
+                  stroke="currentColor"
+                  viewBox="0 0 24 24"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)